modules/mono/csharp_script.cpp

```diff
@@ -410,115 +410,6 @@ ScriptLanguage::ScriptNameCasing CSharpLanguage::preferred_file_name_casing() co
 }
 
 #ifdef TOOLS_ENABLED
-struct VariantCsName {
-	Variant::Type variant_type;
-	const String cs_type;
-};
-
-static String variant_type_to_managed_name(const String &p_var_type_name) {
```

Comment:
This became unused with https://github.com/godotengine/godot/pull/87952 and it's OK to remove it.

Reply:
Thanks for pointing it out, so I know this part of the commit should _not_ be cherry-picked to `4.2`.